app/styles/_components/App/app.sass

```diff
@@ -1,7 +1,7 @@
 #main-container
   flex-grow: 1
   padding-left: $sidebar_width
-  height: calc(100vh - 60px)
+  height: 100%
```

Comment:
This change has an important impact on the video page, and potentially other pages on the site. Could we find a solution that only touches the extension page?

![Peek 2021-01-20 19-35](https://user-images.githubusercontent.com/1556356/105219465-c7109580-5b56-11eb-8098-bfc9844df821.gif)